inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/utils/MetaDataUtils.java

```diff
@@ -47,16 +49,17 @@ public class MetaDataUtils {
     private static final String LOG_MARK = ".log";
 
     // standard log path for k8s
-    private static final String STANDARD_OUT = "/var/log/containers";
+    private static final String FILE_NAME_PATTERN = "(^[-a-zA-Z0-9]+)_([a-zA-Z0-9-]+)_([a-zA-Z0-9-]+)(.log)";
 
     /**
      * standard log for k8s
      *
      * get pod_name,namespace,container_name,container_id
      */
     public static Map<String, String> getLogInfo(String fileName) {
+        Matcher matcher = Pattern.compile(FILE_NAME_PATTERN).matcher(fileName);
```

Comment:
Suggested extracting the `Pattern.compile` to a static final field, it is more efficient.

Reply:
done